src/provider/model/FinEnvironment.ts

```diff
@@ -117,6 +118,12 @@ export class FinEnvironment extends AsyncInit implements Environment {
         }
     }
 
+    public async getEntityType(identity: Identity): Promise<EntityType> {
+        const entityInfo = await fin.System.getEntityInfo(identity.uuid, identity.uuid);
```

Comment:
Is this definitely correct? Not `await fin.System.getEntityInfo(identity.uuid, identity.name);`. Suggest comment if correct, definitely looks like an error